packages/use-contractkit/src/connectors/connectors.ts

```diff
@@ -356,3 +356,85 @@ export class WalletConnectConnector implements Connector {
     return wallet.close();
   }
 }
+
+export class CeloDanceConnector implements Connector {
```

Comment:
Why have you created a new connector? It seems to be just a copy of another connector